src/routes/MyPage.jsx

```diff
@@ -0,0 +1,288 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { SmallPost } from "../components/Posts";
+import {
+  getPosts,
+  getTags,
+  getUserInMyPage,
+  updateUserInMyPage,
+} from "../apis/api";
+import { getCookie } from "../utils/cookie";
+
+const MyPage = () => {
+  const [postList, setPostList] = useState([]);
+  const [profile, setProfile] = useState({
+    id: "",
+    user: {},
+    college: "",
+    major: "",
+  });
+  useEffect(() => {
+    const getPostsAPI = async () => {
+      const posts = await getPosts();
+      setPostList(posts);
+    };
+    getPostsAPI();
+  }, []);
+  const [isEditEmail, setIsEditEmail] = useState(false);
+  const [isEditUsername, setIsEditUsername] = useState(false);
+  const [isEditCollege, setIsEditCollege] = useState(false);
+  const [isEditMajor, setIsEditMajor] = useState(false);
+  const [user, setUser] = useState({});
+  const [email, setEmail] = useState("");
+  const [username, setUsername] = useState("");
+  const [college, setCollege] = useState("");
+  const [major, setMajor] = useState("");
+
+  useEffect(() => {
+    const getProfileAPI = async () => {
+      const myProfile = await getUserInMyPage();
+      console.log(myProfile);
+      setProfile({
+        id: myProfile.id,
+        user: myProfile.user,
+        college: myProfile.college,
+        major: myProfile.major,
+      });
```

Comment:
어차피 백에서 보내주는 형식대로 데이터를 저장할거면 굳이 이렇게 안하구 `setProfile(myProfile)` 이런식으로 하는게 더 편할듯 합니당

그런데 사실 다른 값들은 크게 필요가 없으니 저라면
```suggestion
      setProfile({
        email: myProfile.user.email,
        username: myProfile.user.username,
        college: myProfile.college,
        major: myProfile.major,
      });
```

이런식으로 할거 같아욥